### PR TITLE
Fixes the status bar in the inquiry modal

### PR DESCRIPTION
**Pull request by orta**
Before (on my iPhone X):

img_5aff3618ba01-1

After:

screen shot 2018-10-09 at 11 35 44 am

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ upcoming:
     - Uses metaphysics for the ArtworkVC's Artwork update API - orta
 
   user_facing:
-    - None yet
+    - Fxies to the status bar in an inquiry - orta
```
Comment (ashfurrow) [inline CHANGELOG.yml]: > Fxies

File: Artsy/View_Controllers/Artwork/AREigenInquiryComponentViewController.m [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#import "AREigenInquiryComponentViewController.h"
+
+@implementation AREigenInquiryComponentViewController
+
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+
+    self.view.backgroundColor = [UIColor whiteColor];
```
Comment (alloy) [inline Artsy/View_Controllers/Artwork/AREigenInquiryComponentViewController.m]: Do we want all component VCs coming out of Emission to have white backgrounds by default?

File: CHANGELOG.yml (hunk 1 above)
Comment (alloy) [reply CHANGELOG.yml]: Be gone, hipsters and their fixies! 🚲

**Comment by alloy** [comment]
LGTM 👍 If there’s a specific reason you went with doing this in Eigen rather than Emission, then go ahead and merge.

**Comment by orta** [comment]
Only this VC needs it because most of our VCs are supposed to pull down and have a black BG, this one can't be pulled down etc

**Comment by orta** [comment]
Cool, merge on green

**Comment by alloy** [comment]
Ah gotcha

**Pull request merged**